### DISCUSSION TITLE
Feature Request: allow putting more than 1 free-form URL on GitHub profile page

### DISCUSSION THREAD

**Discussion by HariSekhon** [community/community#28351] [Other Feature Feedback, Questions, & Ideas]
Currently there is only 1 URL website link allowed on a GitHub user's profile page, which I currently point to my LinkedIn profile.

However, I could really use additional optional URL links to point to things like other related sites or portfolio pages.

I could add a link to my DockerHub profile for example.

I also have a large CI/CD web page generated ranking my repos by stars and showing their CI/CD builds at a glance which I would like to add.

In the latter case this is highly relevant not just to seeing a more expanded list of my repos than the pinned repos on my profile page allows (see #28350 about that), but also to see the states of them at a glance.

The use cases for allowing more free-form URLs on a user's GitHub profile page are endless.

**Comment by jfullstackdev** [comment]
if you have a README on your profile, you can customize that, including the extra pins 

just like this 

![image](https://user-images.githubusercontent.com/47092464/182877320-d534c5c9-2c0e-421d-ad50-3086d33fb406.png)
